p11-kit/export-object.c

```diff
@@ -60,12 +60,14 @@ p11_kit_export_object (int argc,
 		       char *argv[]);
 
 static void
-export_x509_certificate (P11KitIter *iter)
+export_attr (P11KitIter *iter,
+	     CK_ATTRIBUTE_TYPE type,
```

Comment:
nit: I would spell `export_attribute` out, as we don't abbreviate `export_certificate` as `export_cert`.